config.md

```diff
@@ -217,6 +217,7 @@ For POSIX platforms the `user` structure has the following fields:
 
 * **`uid`** (int, REQUIRED) specifies the user ID in the [container namespace](glossary.md#container-namespace).
 * **`gid`** (int, REQUIRED) specifies the group ID in the [container namespace](glossary.md#container-namespace).
+* **`umask`** (int, OPTIONAL) specifies the [umask][umask_2] of the user. If unspecified, the umask should not be changed from the calling process' umask.
```

Comment:
nit: the possessive of “process” is “process's”, see us [here][1] and the Linux man pages [here][2].

nit: the “If unspecified…” sentence should [go on its own line][3].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.1/config.md#L149
[2]: https://git.kernel.org/pub/scm/docs/man-pages/man-pages.git/commit/?id=a5a3e91bf97f08652849367eeef112813dceb809
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/style.md#one-sentence-per-line